src/main/g8/build.sbt

```diff
@@ -1,11 +1,13 @@
-import Dependencies._
-
-lazy val root = (project in file(".")).
-  settings(
-    inThisBuild(List(
-      organization := "com.example",
-      scalaVersion := "2.12.1"
-    )),
-    name := "$name$",
-    libraryDependencies ++= Seq(akkaActor, akkaTestkit % Test, scalaTest % Test)
-  )
+name := "hello-akka-scala"
+
+version := "1.0"
+
+scalaVersion := "2.12.2"
+
+lazy val akkaVersion = "2.5.1"
```

Comment:
perhaps we should have g8 parameters for the versions as in https://github.com/akka/akka-http-scala-seed.g8/pull/1 ?